Source/NSFileManager.m

```diff
@@ -685,6 +685,16 @@ - (BOOL) changeFileAttributes: (NSDictionary*)attributes atPath: (NSString*)path
   return allOk;
 }
 
+- (BOOL) setAttributes: (NSDictionary *)attributes
+          ofItemAtPath: (NSString *)path
+                 error: (NSError **)error
+{
+  BOOL success = [self changeFileAttributes:attributes atPath:path];
```

Comment:
I would still add a space after the colons in this line, but the rest looks great to me.